.github/workflows/goalie.yaml

```diff
@@ -8,10 +8,22 @@ on:
   pull_request:
     types: [opened, reopened, synchronize]
 
+permissions:
+  issues: write
+  pull-requests: write
```

Comment:
are these really needed if you use the token from the github app?

Reply:
Yep - so it's twofold. We have the org permissions which is just for members read, and then there's the applying and removing of the labels which is writing to `issues` + `prs`